released/packages/coq-mathcomp-ssreflect/coq-mathcomp-ssreflect.1.7.0/opam

```diff
@@ -2,16 +2,17 @@ opam-version: "1.2"
 name: "coq-mathcomp-ssreflect"
 version: "1.7.0"
 maintainer: "Mathematical Components <mathcomp-dev@sympa.inria.fr>"
+synopsis: "The Mathematical Components library"
 
-homepage: "http://math-comp.github.io/math-comp/"
+homepage: "https://math-comp.github.io/math-comp/"
 bug-reports: "Mathematical Components <mathcomp-dev@sympa.inria.fr>"
 dev-repo: "https://github.com/math-comp/math-comp.git"
 license: "CeCILL-B"
 
-build: [ make "-C" "mathcomp/ssreflect" "-j" "%{jobs}%" ]
-install: [ make "-C" "mathcomp/ssreflect" "install" ]
+build: [ make "-C" "mathcomp/ssreflect" "-C" "mathcomp/ssreflect" "-j" "%{jobs}%" ]
+install: [ make "-C" "mathcomp/ssreflect" "-C" "mathcomp/ssreflect" "install" ]
```

Comment:
That double `-C` looks weird, doesn't it?

Reply:
right... this is automatically generated, I should doublecheck the generator :smile: